lib/flavor.js

```diff
@@ -43,6 +46,17 @@ export const cantAffordWords = [
   'need doubloons...',
   'too pricey...',
   // "when you're richer...",
+  'beyond your means...',
+  'not in your budget...',
+  "you can't afford that...",
+  'too expensive...',
+  // "can't afford this...",
+  "can't afford...",
+  'unaffordable...',
+  'out of reach...',
+  'need doubloons...',
+  'too pricey...',
+  // "when you're richer...",
```

Comment:
Have you tested this? We shortened these pretty aggressively because the buttons are so small

Reply:
![image](https://github.com/user-attachments/assets/df7772d7-de2d-4ac9-a042-ca37d4aa687e)

Case in point